### ISSUE TITLE
Tab numbers position problem at some zoom levels

### ISSUE THREAD

**Issue by thoun** [CoderLine/alphaTab#452]
# Your environment

* Version used: develop / demo on the website
* Platform used: JavaScript
* Rendering engine used: SVG
* Browser Name and Version: Chrome 86 & Firefox too
* Operating System and version (desktop or mobile): Windows 10
* Link to your project: visible on https://alphatab.net/ demo page

# Expected Results
Numbers are vertically centered to line on every zoom level

# Observed Results
numbers are not aligned at 75% (ok with 50% and 90%)
![atissue](https://user-images.githubusercontent.com/26171303/101283876-88f52780-37dd-11eb-9028-436213258521.png)

# Steps to Reproduce (for bugs)
Open https://alphatab.net/ and select 75% zoom

# Possible Solution 
No solution yet. I might try a MR if I find tome to look deeper to this bug.

**Comment by Danielku15** [comment]
The issue likely comes from the way the numbers are vertically aligned. To maintain compatibility with IE I had to change the way texts are vertically aligned. Instead of the old `dominant-baseline="middle"` or `dominant-baseline="hanging"` the centering works via `dy="0.5ex"` or `dy="1.25ex"`. 

With the latest market changes it might be worth reconsidering again the IE support and drop it. Bootstrap 5 will drop support, Microsoft will drop support in August 2021 for their M365 apps, Wordpress is considering it. I opened a [discussion](https://github.com/CoderLine/alphaTab/discussions/460) to find out if we could drop it. 

Until then we either are able to solve it using the existing `ex` units or we could try to add some browser detection to use `dx` units only for IE.
